src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,10 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
```

Comment:
Please don't add redundant empty lines to your code, such as after the class declaration or method signature.